src/api/merchants/merchant-configs.md

```diff
@@ -24,11 +24,18 @@ A Merchant Config defines the available payment options for paying a [Payment Re
 
 {% h4 Required Fields %}
 
-| Field          | Type   | Description                   |
-| :------------  | :----- | :---------------------------  |
+| Field          | Type   | Description                          |
+| :------------  | :----- | :----------------------------------- |
 | paymentOptions | Array  | Array of [Payment Option Configs][]. |
 
 
+{% h4 Optional Fields %}
+
+|    Field            | Type  |                                               Description                                                            |
+| :------------------ | :---- | :------------------------------------------------------------------------------------------------------------------- |
+| allowedRedirectUrls | Array | **Experimental** List of allowewd url prefixes that the user can go to after paying or cancelling a payment request. |
```

Comment:
I'm thinking this is not going to scale well. A URL is generally inclusive of a resource.

i.e. redirects might go back to something dynamic like https://app.cp42.click/pay/3923067a-5e4e-421b-952b-3e915e2b43d3, and specifying all URLs is a bit excessive.

Are we allowing for more general redirects like https://app.cp42.click/hello/*?
If so, can we rename this to something more general like allowedRedirectFQDNs, or allowedRedirectHosts?

Reply:
I like allowedRedirectHosts

Reply:
There's a small type allowewd --> allowed

Reply:
@heymatthew It is a prefix. I thought "allowedRedirectUrlPrefixes" was too wordy so settled on "allowedRedirectUrls". If you read the description it tells you that it is a prefix.